### ISSUE TITLE
Introduce checks when adding new payment accounts

### ISSUE THREAD

**Issue by godfuture** [haveno-dex/haveno#1084]
I have added my SEPA payment account lately. I filled all details and saved.

When I started my very first trade, Haveno UI has given me the hint that I should check for exact name and IBAN otherwise I should not accept. In this moment I noticed I had an issue. My name contains german special character, and banks normally replace "ä" to "ae" and so on. If I would send money to buy XMR, there wouldnt be a 100% match of my name in Haveno compared to banking transaction details.

Please add checks to payment details when adding payment accounts.

As said, first check to be added as part of this ticket could be the SWIFT latin character set in SEPA bank account owner field. But I guess there are more useful checks in other payment methods as well.

Maybe this could be start of something...just sayin...

**Comment by boldsuck** [comment]
If so, the check should only show a hint and not enforce the latin character set.

Why:
Account age and signing status is tied to a hash. You can't change anything afterwards.
This would reset all accounts currently signed & account age is reset to 0.
I have already signed several accounts whose user names have Turkish, Scandinavian or French accents.
People have invested money for signing and would not be happy if they had to start from scratch again.

**Comment by godfuture** [comment]
Agreed. It was unclear to me if an user can claim the transaction to be false, because the names are not 100% matching. Is this a real problem also considering the case where arbitrators need to decide during a claim where users are afraid of wrong data given by counterpart?

If not, maybe it is enough to change the automatic hint haveno is giving to users. Maybe we can write that the name has to be the same but everyone should consider the possible differences through latin char set?

**Comment by boldsuck** [comment]
I can say from experience that there is not always a 100% match. Today I had someone with two first names in the bank transfer but only one in Haveno.
Generally, this is intended as fraud protection for you. As protection against chargebacks for stolen bank accounts. Payer name, account numbers, payment method¹ & institution should match. It doesn't matter whether it says Bob Müller or Bob Mueller.
I'll make this issue known to reto arbs in the Simplex chat.

¹It is common for a SEPA trade to be paid with SEPA Instant. Name and IBAN must of course match.

**Comment by godfuture** [comment]
> It doesn't matter whether it says Bob Müller or Bob Mueller.
> I'll make this issue known to reto arbs in the Simplex chat.

Understood. In this case, maybe haveno could be adjusted to explain that there are expected differences and unexpected. This will make the usage of the app even clearer. 

Thanks!
